woocommerce/woocommerce
# Update concurrency group for community label workflow

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).

### Changes proposed in this Pull Request:


Closes https://github.com/woocommerce/woocommerce/issues/47179

Updated the concurrency group for the `community-label.yml` workflow to use an unique group. The github.ref is always returning `refs/heads/trunk` incorrectly cancelling some runs.

I also checked for other workflows using `issues` or `pull_request_target` triggers but no other was using concurrency.

### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

After merge, open two issues at the same time. The `Add Community Label, Assign Reviewers` should run for both, the first run should not be cancelled by the second one.
Repeat the same test with two pull requests.
